app/src/Components/Login/Login.js

```diff
@@ -1,35 +1,34 @@
 import React from 'react';
```

Comment:
Revert formatting changes to prevent future merge conflicts

Reply:
See 0aafaafba016b3ecb6d62307026422341f1e620a